### DISCUSSION TITLE
Buffer window customization

### DISCUSSION THREAD

**Discussion by encomun** [nobiot/org-remark#43] [Q&A] [answered]
I'm really excited about using this package. It's just what I need for writing and editing. 

I'm new to emacs and not very knowledgeable. I would like the notes buffer to open to the right instead of the left. How would I make this customization? The manual names 'org-remark-notes-display-buffer-action' but I'm not sure what exactly to add to my init file. 

Sorry for the simple question, but I would greatly appreciate the assistance.

**Comment by nobiot** [comment, marked as answer]
If you are new to Emacs, I'd suggest you try Customize. `M-x customize` and search for `org-remark-notes-display-buffer-action`

You can just change "left" to "right" and press Apply. This lasts until you quit Emacs. If you press "Apply and Save", the change gets persisted, and next time you launch Emacs, it's still there.

![image](https://user-images.githubusercontent.com/12507865/204057412-189eaaa4-f319-4038-819b-a7f439a1ea71.png)

If you must use `setq`, you can do the following
````
(setq org-remark-notes-display-buffer-action
  `((display-buffer-in-side-window)
    (side . right)
    (slot . 1)))
```

**Reply by encomun** [reply]
Thank you so much for your quick reply. I appreciate the help. 

And thanks for this great tool.

**Reply by nobiot** [reply]
You’re welcome ☺️
